config/config-local.yaml

```diff
@@ -0,0 +1,13 @@
+app:
+  debug: true
+
+logger:
+  level: DEBUG
+  base_name: multi-api
+
+db:
+  # docker
+  # host: db
```

Comment:
Do u need this commented code ? If not , remove commented code

Reply:
yes, i need
because if you to run in locally in docker you need to change host to db

Reply:
and u will always keep commented code for it ?

Reply:
yes, until I add it into readme

Reply:
i think it's not good approach to keep commented code. U have git diff for it